seps/sep-0002.md

```diff
@@ -0,0 +1,225 @@
+---
+SEP: 2
+Title: WIP Standardize Groups
+Authors: Charles G. <charles@near.org>
+Status: Draft
+DiscussionsTo: https://github.com/nearsocial/SEPs/pull/2
+Version: 1.0.0
+Created: 2023-10-15
+Last Updated: 2023-10-15
+---
+
+
+## Summary
+
+This document serves to come to an agreement on a specification for a new "Groups" experience on Near Social, designed to allow people to create and discover communities around specific topics, interests, projects, governance, etc.
+
+## Motivation
+
+We need a SocialDB standard for groups to support discoverability on near.org and interoperability between gateways and dApps to mitigate the [compatibility problems](https://near.social/devgovgigs.near/widget/gigs-board.pages.Post?id=1034) which arise when dApps build distinct implementations of communities.
+
+
+## UX Goals
+
+1. To allow a group of users to moderate with sovernity a scope of socialDB limited to the members of the group. e.g. group members have read + write access to the group's socialDB namespace. 
+1. Non-members only have read-only access to view group content
+1. Group members should receive notifications for all relevant social interactions in a group (e.g. Like, Reply for Posts and Discussions)
+1. All content posted to groups that a user is part of should appear the user’s "Following" activity feed. 
+1. Leaving a group will stop content from appearing on a user’s feed.
+1. Group content should only appear in the “All” activity feed if a group member reposts a group post to the main feed.
```

Comment:
@charleslavon This is not an action that is currently supported today on near.org but it does exist on near.social. Is the idea that a user will be able to select where they can repost content? Is the idea here that a user would be able to select where they can repost content? This is probably out of scope for this discussion around groups so feel free to resolve and we can discuss at a later time.